src/containers/UpdatePassword/UpdatePassword.js

```diff
@@ -0,0 +1,180 @@
+import React, { useState } from "react";
+import Avatar from "@material-ui/core/Avatar";
+import Button from "@material-ui/core/Button";
+import CssBaseline from "@material-ui/core/CssBaseline";
+import Box from "@material-ui/core/Box";
+import LockOutlinedIcon from "@material-ui/icons/LockOutlined";
+import Typography from "@material-ui/core/Typography";
+import Container from "@material-ui/core/Container";
+import { makeStyles } from "@material-ui/core/styles";
+import TextField from "@material-ui/core/TextField";
+import Copyright from "../../components/CopyRight";
+
+const useStyles = makeStyles((theme) => ({
+  resetPassword: {
+    marginTop: theme.spacing(8),
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+  },
+  avatar: {
+    margin: theme.spacing(1),
+    backgroundColor: theme.palette.secondary.main,
+  },
+  form: {
+    width: "100%",
+    marginTop: theme.spacing(1),
+  },
+  submit: {
+    margin: theme.spacing(3, 0, 2),
+  },
+  label: {
+    fontSize: "17px",
+  },
+  password: {
+    fontSize: "17px",
+    margin: theme.spacing(3, 0, 2),
+    marginLeft: theme.spacing(13),
+  },
+  confirmPassword: {
+    fontSize: "17px",
+    margin: theme.spacing(3, 0, 2),
+    marginLeft: theme.spacing(5),
+  },
+  error: {
+    margin: theme.spacing(3, 0, 2),
+    color: "red",
+  },
+  success: {
+    margin: theme.spacing(3, 0, 2),
+    color: "green",
+  },
+}));
+
+function UpdatePassword() {
+  const [oldPassword, setOldPassword] = useState("");
+  const [newPassword, setNewPassword] = useState("");
+  const [confirmNewPassword, setConfirmNewPassword] = useState("");
+  const [error, setError] = useState(null);
+
+  const updateOldPassword = (e) => {
+    setOldPassword(e.target.value);
+  };
+  const updateNewPassword = (e) => {
+    setNewPassword(e.target.value);
+  };
+  const updateConfirmNewPassword = (e) => {
+    setConfirmNewPassword(e.target.value);
+  };
+
+  const checkValues = () => {
+    if (newPassword === "" || confirmNewPassword === "" || oldPassword === "") {
+      setError("CAN NOT BE EMPTY FIELD!");
```

Comment:
Great work, I just thinking that you should not make error messages all caps like these.

Reply:
@AbdullahMuhammed5 I think, it's for UX purposes, just to be more noticed by user attention! if the other fellows don't like it, we can revert it back! when we combine all `USER SIGNUP/LOGIN` components together :+1: